Translations/zh-Hans.lproj/Localizable.strings

```diff
@@ -1,39 +1,39 @@
 "_BASE_USED_" = "zh-Hans";
-"_ADVANCED_EXTRA_WIDTH_" = "10.0";
+"_ADVANCED_EXTRA_WIDTH_" = "0.0";
 "_PREFERENCES_EXTRA_WIDTH_" = "0.0";
 "OK" = "好";
 "Yes" = "是";
 "No" = "否";
 "Don't show again" = "不再提示";
 "Set as default compressor" = "设置为默认压缩工具";
-"Not now" = "现在不";
-"Want to make Keka your default compressor?" = "想要使用 Keka 成为您的默认压缩工具?";
-"This means that all the compatible files will be extracted with Keka by now. You can set this later in the Keka Preferences." = "这意味着现在所有兼容的文件将被 Keka 解压. 您稍后可以在 Keka 首选项中设置.";
-"Really want to close Keka?" = "确认关闭 Keka?";
-"The current operations will be canceled. If you choose to wait and quit Keka will exit when all work is done." = "当前操作将被取消. 如果你选择等待并退出, Keka 将在所有的工作完成后退出.";
+"Not now" = "暂时不要";
+"Want to make Keka your default compressor?" = "将 Keka 设置为您的默认压缩工具？";
+"This means that all the compatible files will be extracted with Keka by now. You can set this later in the Keka Preferences." = "这意味着现在所有兼容的文件都被 Keka 解压缩。您稍后可以在 Keka 的偏好设置中设置。";
+"Really want to close Keka?" = "确认关闭 Keka？";
+"The current operations will be canceled. If you choose to wait and quit Keka will exit when all work is done." = "当前操作将被取消。如果您选择等待并退出，Keka 将在所有工作完成后退出。";
 "Password needed" = "需要密码";
-"File \"%@\" needs a password to be extracted" = "解压 \"%@\" 需要密码";
-"Extraction of \"%@\" done" = "解压\"%@\" 完成";
-"Extraction of \"%@\" failed." = "解压 \"%@\" 失败.";
-"Extraction of %i files done" = "解压 %i 文件完成";
+"File \"%@\" needs a password to be extracted" = "解压缩“%@”需要密码";
+"Extraction of \"%@\" done" = "解压缩“%@”完成";
+"Extraction of \"%@\" failed." = "解压缩“%@”失败。";
+"Extraction of %i files done" = "解压缩 %i 文件完成";
 "Extraction complete" = "解压缩完成";
 "Extraction fail" = "解压缩失败";
 "Compression complete" = "压缩完成";
-"File \"%@\" created" = "文件 \"%@\" 已创建";
+"File \"%@\" created" = "已创建“%@”文件";
 "Compression fail" = "压缩失败";
-"File \"%@\" cannot be created" = "文件 \"%@\" 不能被创建";
+"File \"%@\" cannot be created" = "不能创建“%@”文件";
 "Operation not implemented" = "操作未执行";
-"This operation is not implemented yet so Keka is in beta process." = "这个操作还没有执行因为 Keka 正处于测试阶段.";
-"Performing operation..." = "正在执行操作...";
-"Waiting..." = "请稍候...";
-"Extracting \"%@\"" = "正在解压 \"%@\"";
-"Extracting \"%@...\"" = "正在解压 \"%@...\"";
-"Creating %@ file..." = "创建 %@ 文件中...";
-"Creating file \"%@\"" = "正在创建文件 \"%@\"";
-"Creating file \"%@...\"" = "正在创建文件 \"%@...\"";
-"Time elapsed: %d:%@%d:%@%d hours%@" = "已用时间: %d:%@%d:%@%d 小时%@";
-"Time elapsed: %d:%@%d minutes%@" = "已用时间: %d:%@%d 分钟%@";
-"Time elapsed: %d seconds%@" = "已用时间: %d 秒%@";
+"This operation is not implemented yet so Keka is in beta process." = "此操作还没有执行，因为 Keka 正处于测试阶段。";
+"Performing operation..." = "正在执行操作……";
```

Comment:
Those must be three single dots. You're right about the dot size (smaller) but don't let it convert to an ascii character.